Assets/GothicVR/Scripts/Player/Menu/MenuManager.cs

```diff
@@ -0,0 +1,87 @@
+using GVR.Creator;
+using GVR.Manager;
+using GVR.Phoenix.Util;
+using UnityEngine;
+
+namespace GVR.Player.Menu
+{
+    public class MenuManager : MonoBehaviour
+    {
+        public GameObject MainMenu;
+        public GameObject SettingsMenu;
+        public GameObject TeleportMenu;
+        public GameObject MovementMenu;
+        public GameObject UIMenu;
+
+        void Awake()
+        {
+            SetSettingsValues();
+        }
+
+        public void SetSettingsValues()
+        {
+            var moveSetting = MovementMenu.FindChildRecursively("MoveSpeedSlider").GetComponent<MoveSpeedController>();
+            moveSetting.ChangeMoveSpeed(PlayerPrefs.GetFloat(ConstantsManager.I.moveSpeedPlayerPref));
+
+            var turnSetting = MovementMenu.FindChildRecursively("TurnSettingDropdown").GetComponent<TurnSettingDropdownController>();
+            turnSetting.DropdownItemSelected(PlayerPrefs.GetInt(ConstantsManager.I.turnSettingPlayerPref));
```

Comment:
Can we just assign these values from Scene view? Would be more performant and transparent for coders.

Reply:
This is meant to load the saved values when loading the VR Player ingame. If you want to change settings in the main menu, we want them to reflect when in-game)